Source/CarthageKit/Xcode.swift

```diff
@@ -1283,7 +1283,7 @@ private func stripArchitecture(frameworkURL: NSURL, architecture: String) -> Sig
 }
 
 /// Returns a signal of all architectures present in a given framework.
-public func architecturesInFramework(frameworkURL: NSURL) -> SignalProducer<String, CarthageError> {
+private func architecturesInFramework(frameworkURL: NSURL) -> SignalProducer<String, CarthageError> {
```

Comment:
Ah ok—was unaware of whether specs had access to private methods. I'll move it back to public.